vmc/resource_vmc_cluster.go

```diff
@@ -155,7 +155,8 @@ func resourceCluster() *schema.Resource {
 
 				if d.Get("storage_capacity").(string) == "" {
 
-					return fmt.Errorf("storage_capacity is required for host_instance_type %q", newInstanceType)
+					return fmt.Errorf("storage_capacity is required for host_instance_type %q "+
+						"Possible values are 15TB, 20TB, 25TB, 30TB, 35TB per host", newInstanceType)
 
```

Comment:
You seemed to have removed `storage_capacity` variable from the .tf files above, are we still supporting this?